models/DataObject/ClassDefinition/Data/UrlSlug.php

```diff
@@ -25,7 +25,7 @@
 use Pimcore\Model\DataObject\Concrete;
 use Pimcore\Model\DataObject\Fieldcollection\Data\AbstractData;
 use Pimcore\Model\DataObject\Localizedfield;
-use Pimcore\Model\Redirect;
+use Pimcore\Bundle\SeoBundle\Model\Redirect;
```

Comment:
What happens if the bundle ins not installed and there are slugs? 
This will break the save function right? 

Maybe we can move this to a save listener instead?

Reply:
What about this one? :)